src/main/java/org/trivetia/mace/common/CommonProxy.java

```diff
@@ -0,0 +1,39 @@
+package org.trivetia.mace.common;
+
+import java.io.File;
+import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import org.trivetia.mace.config.MaceConfigs;
+
+/**
+ *
+ * @author Dj
+ */
+@Mod.EventBusSubscriber
+public class CommonProxy 
+{
+    // Config instance
+    public static MaceConfigs maceCfgs = null;
+    
+    public void preInit(FMLPreInitializationEvent e) 
+    {
+        /*File directory = e.getModConfigurationDirectory();

```

Comment:
Remove the code if it is not needed or add a comment explaing why it is still here